tests/negative/regression/filesystem.go

```diff
@@ -25,8 +25,7 @@ func init() {
 
 func VFATIgnoresWipeFilesystem() types.Test {
 	// Originally found in https://github.com/coreos/bugs/issues/2055
-
-	name := "Regression: VFAT ignores WipeFilesystem"
+	name := "regression.fs.ignoreswipe"
```

Comment:
This PR seems to switch between `fs` & `filesystem` in test names. Is your intention that it's spelled out only if it's in the top level?